src/components/Input/Input.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import './Input.css';
+
+type InputProps = {
+  value: string;
+  onChange: React.ChangeEventHandler;
```

Comment:
лучше в этот компонент передавать
```suggestion
  onChange: (value: string) => void;
```

и внутри `Input` создать `const handleChange = (e) => onChange(e,target.value)`
чтобы в компонентах вне, которые используют `Input`, не писать постоянно `e.target.value`